axelrod/strategies/axelrod_second.py

```diff
@@ -199,3 +202,201 @@ def strategy(self, opponent: Player) -> Action:
+            Where JA and J are the last moves of the strategy and the opponent (C=0, D=1)                                                                                                                                                                       
+            Increase the value of NK by 1                                                                                                                                                                                                                               
+            Cooperate                                                                                                                                                                                                                                                   
+                                                                                                                                                                                                                                                                        
+    Tranquilizer came in 27th place in Axelrod's second torunament. 
+   
+
+    Names:
+
+    - Craig Feathers: [Axelrod1980]_
+    - Tranquilizer: [Axelrod1980]_
+    """
+
+    name = 'Tranquilizer'
+    classifier = {
+        'memory_depth': float('inf'),
+        'stochastic': True,
+        'makes_use_of': {"game"},
+        'long_run_time': False,
+        'inspects_source': False,
+        'manipulates_source': False,
+        'manipulates_state': False
+    }
+
+    # Initialised atributes
+    def __init__(self):
+        super().__init__()
+        self.FD = 0
+        self.consecutive_defections = 0
+        self.ratio_FD1 = 5
+        self.ratio_FD2 = 0
+        self.ratio_FD1_count = 1
+        self.ratio_FD2_count = 1
+        self.score = None
+        self.P = 1.1
+        self.current_score = 0
+        self.dict = {C: 0, D: 1}
+
+
+    def update_stateFD(self, opponent):  
+        
+        """
+        Calculates the ratioFD values and P values, as well as sets the 
+        states of FD at the start of each turn
+        """
+        self.current_score = compute_final_score(zip(self.history, opponent.history))
+
+        if self.FD == 2:
+            self.FD = 0
+            self.ratio_FD2 = (self.ratio_FD2 * self.ratio_FD2_count)
+            self.ratio_FD2 += 3 - (3 * self.dict[opponent.history[-1]]) 
+            self.ratio_FD2 += (2 * self.dict[self.history[-1]]) 
+            self.ratio_FD2 -= self.dict[opponent.history[-1]] * self.dict[self.history[-1]]
+            self.ratio_FD2 /= (self.ratio_FD2_count + 1)
+            self.ratio_FD2_count += 1
+        elif self.FD == 1:
+            self.FD = 2
+            self.ratio_FD1 = ((self.ratio_FD1 * self.ratio_FD1_count)
```

Comment:
There's a syntax error here. You have opened two brackets and only closed one.

It's probably always a good idea to run your tests locally (even if you only run the strategy specific ones) to make sure small typos like this don't get in :) (They get picked up by the automated test runners anyway so it's not a big deal but it'll help you get feedback faster :)) :+1: